src/main/resources/com/google/cloud/spanner/jdbc/DatabaseMetaData_GetCrossReferences.sql

```diff
@@ -1,3 +1,4 @@
+/*GSQL*/
 /*
  * Copyright 2019 Google LLC
```

Comment:
Question: Usually I've observed Copyright comments to be the first line / comment in code bases. Is it ok to have the `/*GSQL*/` as the first line instead, given its trivial nature?

Reply:
Yes, there is no requirement that the copyright header is the outright first part of a source file (although it is common practice). See also https://opensource.google/documentation/reference/copyright

Reply:
Thanks, this is helpful.